views/administration/users_edit.hbs

```diff
@@ -61,82 +61,96 @@
-                    <label style="margin-bottom: 20px"><input name="parent_termsOfUseConsent" type="checkbox"
-                            value="true" disabled {{#if consent.parentConsent.termsOfUseConsent}}checked{{/if}} />
-                        {{$t "administration.label.termsOfUse" }}</label><br />
-                </div>
-                <div class="col-md-6">
-                    <b>{{$t "administration.global.label.student" }}</b><br />
-                    <input name="student_consentId" type="hidden" value="{{consent._id}}" disabled />
-
-                    <label style="margin: 15px 10px 10px 0"><input name="student_form" value="analog" type="radio"
-                            disabled {{#ifeq consent.userConsent.form 'analog'}}checked{{/ifeq}} />
-                        {{$t "administration.label.analog" }}</label>
-                    <label><input name="student_form" value="digital" type="radio" disabled
-                            {{#ifeq consent.userConsent.form 'digital'}}checked{{/ifeq}} />
-                        {{$t "administration.label.digital" }}</label>
-                    <br />
-                    <label><input name="student_privacyConsent" type="checkbox" value="true" disabled
-                            {{#if consent.userConsent.privacyConsent}}checked{{/if}} />
-                        {{$t "administration.global.label.dataProtection" }}</label><br />
-                    <label style="margin-bottom: 20px"><input name="student_termsOfUseConsent" type="checkbox"
-                            value="true" disabled {{#if consent.userConsent.termsOfUseConsent}}checked{{/if}} />
-                        {{$t "administration.label.termsOfUse" }}</label><br />
-                </div>
-                {{else}}
-                <div class="col-md-6">
-                    <p>
-                        {{$t "administration.longText.text.thereIsNoDeclarationOfConsent" }}
-                    </p>
-                    <p>
-                        <a class="btn btn-secondary" href="/administration/students/{{user._id}}/skipregistration">
-                            <i class="fa fa-check-square-o"></i>
-                            {{$t "administration.button.receivedConsentOnPaper" }}
-                        </a>
-                    </p>
+            {{#ifCond cloudConsentNecessary '||' editTeacher}}
+                {{#unless @root.hasImportHash}}<button class="btn btn-sm btn-secondary pull-right" id="edit-consent"
+                    type="button"> <i style="margin-right: 3px" class="fa fa-edit"></i>
+                    {{$t "global.button.edit" }}</button>{{/unless}}
+                <h2 class="h5">{{$t "administration.button.consent" }} {{{@root.consentStatusIcon}}}</h2>
+                <div class="row" id="consents-overview">
+                    {{#unless @root.editTeacher}}
+                    {{#unless @root.hasImportHash}}
```

Comment:
indentation needed for better readability on where which condition ends

Reply:
Absolutely agree and added indendations on nested conditions